core/evalir.ml

```diff
@@ -678,8 +678,8 @@ struct
            | None -> computation env cont e
            | Some (db, p) ->
              begin
-               if db#driver_name() <> "postgresql"
-                 then raise (Errors.Runtime_error "Only PostgreSQL database driver supports shredding");
+               if not (List.mem (db#driver_name()) ["postgresql"; "sqlite3"])
```

Comment:
I wonder if we should instead add a test to `db` objects that tells us whether the db supports shredding, and report "this database driver XYZ does not support shredding".